addons/TextAudio/src/style.css

```diff
@@ -70,6 +70,16 @@
     cursor: pointer;
 }
 
+.textaudio-volume-container {
+    display: flex;
+    flex-direction: row;
+    width: 50px;
+}
+
+.textaudio-volume-container.keyboard_navigation_active_element.active {
+    width: 175px;
```

Comment:
Obawiam się że nowe style mogą popsuć widoki istniejących lekcji. Raczej button nikt nie będzie powiększać powyżej 50px, ale "textaudio-volume-layer" teoretycznie mógłby względem domyślnej wartości. A tym samym 175px może być niewystarczające dla obu elementów.

Proponował bym nie ustawianie szerokości i/lub prośba o testy na kilku istniejących lekcjach.
Jeżeli nie ustawi się szerokości to wizualnie nie będzie domyślnie dobrze wyglądać, ale także nie powinno popsuć istniejących lekcji. Co myślisz ?